core/src/main/java/com/eaglegenomics/simlims/core/Note.java

```diff
@@ -160,4 +160,10 @@ else if (getText() != null && o.getText() != null) {
     }
     return 0;
   }
+// TODO remove me after sonarcloud does its thing
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)